src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1524,8 +1524,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
     }
 
     void visit_ListItem(const ASR::ListItem_t& x) {
-        ASR::ttype_t* el_type = ASRUtils::get_contained_type(
-                                        ASRUtils::expr_type(x.m_a));
+        ASR::ttype_t *el_type = ASRUtils::get_contained_type(ASRUtils::expr_type(x.m_a));
```

Comment:
Let's not do this formatting change. It passes the `80` character limit in editors/terminal. The previous one seems better.